anime_downloader/sites/helpers/request.py

```diff
@@ -48,7 +49,19 @@ def setup_func(url: str,
         referer : str
             a url sent as referer in request headers
         '''
-        sess = cf_session if cf else req_session
+        selescrape = 'dummy message'
```

Comment:
wot is this?

Reply:
because undefined error on line 81 if the import fails iirc

Reply:
'dummy message' is not acceptable. Put it as `None` or something